README.md

```diff
@@ -1,141 +1,146 @@
-## 4.1 Price Feed Oracle
-Agents can agree to act as price feed oracles by collateralizing an agreement with ANT that commits them to provide accurate price information and not attempt to manipulate the results of the feed. We can take the median value from all participants weighted by their collateral to get a price for a specific period.
+**Redistributing Collateral**
 
-If an agent tries to manipulate the price feed, a dispute can be created and the agent's collateral taken. Since price information is widely available and fairly objective, this mechanism should provide reasonably strong guarantees under the assumption there is at least one honest agent monitoring the price feed.
+If there was any appeal rounds before the final ruling, the total amount of collateral that was deposited for triggering each round will be assigned to the appealer supporting the final ruling.
 
-Agents which provide the price feed take on risk by collateralizing an agreement and must be compensated both for that risk and for providing a service to the network. A portion of inflation is dynamically allocated to incentivizing this behavior based on a target level of participation.
-## 4.2 Stability Reserve
-The purpose of the stability reserve is to dampen the volatility of ANT, particularly on the downside, in order to ensure that it can be effectively used as a unit of account and store of value for collateralizing agreements. By optimizing for store of value and unit of account properties, it is likely that ANT could also become a common medium of exchange.
+In the first adjudication round of a dispute (the original round created with the dispute), the Court doesn't directly manage collateral (as there may not be collateral at all or a different token is used). In case that there was collateral at stake depending on the Court ruling, whenever the Court has a final ruling it will notify the contract being *arbitered* with the ruling and this contract can then distribute collateral.
 
-As the demand for ANT grows and prices rise, ANT is minted and sold to collateralize a reserve. When prices fall the reserve automatically sells collateral in order to buy ANT and remove it from circulation. If there is insufficient collateral in the reserve to buy ANT and maintain price targets, bonds can be issued to support the price and paid back by the network at a later date. Unlike other bond mechanisms, the Network's ability to pay back bonds in the future is not strictly tied to appreciation of ANT, but also from revenue generated by services provided by the network such as the Court.
+## Fee Summary
```

Comment:
Probably for another PR, but it would be good to have more detail about all fees here, maybe with some concrete examples with numbers, from all the perspectives: proposer and appealer (winner/loser), jurors on the winning side, jurors on the losing side, etc.

Reply:
Agreed, we should continue to iterate on this as we go.